Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM node:8.9.4-alpine
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.4
```

Comment:
don't change this, its only for java.
For now update it to the latest 8.x nodejs.
After the october 1st rush, an upgrade to node 10 should be done